index.html

```diff
@@ -1,9 +1,23 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>Noodles!</title>
-</head>
-<body>
-  <h1>Noodles</h1>
-</body>
+  <head>
+    <title>Top 4 Noodle Recipes</title>
+  </head>
+  <body>
+    <h1>Top 4 best noodle recipes for the whole family</h1>
+    <div>
+      <img src="noodlebanner.jpeg" alt="Banner with noodles" width="650"><hr/></div>
+    <div>
+    <p>
+      <h2>Chicken Noodles</h2><a href="noodlesoup.html">Chicken Noodles Soup</a><br>
```

Comment:
usually we would use the `section` tag or the `div` tag here instead of the `p` tag. `p` tags usually consists of just text rather than a header element